etl-repair.md

```diff
@@ -69,6 +69,19 @@ $ kubectl logs deploy/kubecost-cost-analyzer | grep CloudCost
 ```
 {% endcode %}
 
+By default, all users v1.106+ no longer have CloudUsage ETL enabled (`.Values.kubecostModel.etlCloudUsage`, `.Values.kubecostModel.etlCloudAsset`). If you are using CloudUsage ETL, use the commands below:
```

Comment:
The wording feels a bit off after the comma. 

How about something like: 
```suggestion
By default, Kubecost v1.106+ no longer has CloudUsage ETL enabled (`.Values.kubecostModel.etlCloudUsage`, `.Values.kubecostModel.etlCloudAsset`). If you are using CloudUsage ETL, use the commands below:
```

Reply:
I like it!